.gitignore

```diff
@@ -34,3 +34,4 @@ debug
 resource.syso
 
 *.resource.go
+bindata.go
```

Comment:
okay so i think egon had in mind that we would name in something.resource.go

Reply:
Ok, I didn't get that.